recipes/newrelic/infrastructure/windows.yml

```diff
@@ -26,63 +26,26 @@ install:
   tasks:
     default:
       cmds:
-        - task: remove_any_previous
         - task: install
         - task: assert_agent_started
 
-    remove_any_previous:
```

Comment:
@noahmmcgivern we need to keep this.
The Windows infra agent needs to be un-installed (if it exists) before installing a new one.